ui_framework/components/expression/_expression.scss

```diff
@@ -0,0 +1,107 @@
+$expressionColorHighlight: #00A69B;
+
+.kuiExpression {
+
+  .kuiExpression__item {
+    display: inline-block;
+    margin-right: 10px;
+    position: relative;
+  }
+
+  .kuiExpression__button {
+    background-color: transparent;
+    padding: 5px 0px;
+    border: none;
+    border-bottom: dotted 2px $globalBorderColor;
+    font-size: $globalFontSize;
+    cursor: pointer;
+
+    .kuiExpression__buttonDescription {
+      color: $expressionColorHighlight;
+      text-transform: uppercase;
+    }
+
+    .kuiExpression__buttonValue {
+      color: $globalTextColor;
+      text-transform: lowercase;
```

Comment:
I think we can remove this rule, since I think it's reasonable to assume we may want to use values with capital letters, and it should be trivial to use JS to format the value with all lower-case, if need be.